label_sleuth/config.json

```diff
@@ -1,6 +1,6 @@
 {
-    "first_model_positive_threshold": 20,
-    "changed_element_threshold": 20,
+    "first_model_positive_threshold": 4,
+    "changed_element_threshold": 4,
```

Comment:
please keep the original thresholds. thanks :)

Reply:
same as above